types/esri-leaflet/index.d.ts

```diff
@@ -596,6 +607,150 @@ declare module 'leaflet' {
         type FeatureCallbackHandler = (error?: any, featureCollection?: any, response?: any) => void;
         type ResponseCallbackHandler = (error?: any, response?: any) => void;
 
+        /**
+         * Options for `L.esri.ImageMapLayer`
+         */
+        interface ImageMapLayerOptions extends L.ImageOverlayOptions, LayerOptionsBase {
+            /**
+             * If there are multiple bands, you can specify which bands to export
+             */
+            bandIds?: string | undefined;
+
+            /**
+             * The pixel value representing no information.
+             */
+            noData?: number | undefined;
+
+            /**
+             * Interpretation of the noData setting.
+             */
+            noDataInterpretation?: string | undefined;
+
+            /**
+             * 	Leave pixelType as unspecified, or UNKNOWN, in most exportImage use cases, unless such pixelType is
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/esri-leaflet/index.d.ts;
  Line: 630;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.